A80-pid.md

```diff
@@ -0,0 +1,312 @@
+A68: PID LB policy.
+----
+* Author(s): @s-matyukevich
+* Approver: 
+* Status: Draft
+* Implemented in: PoC in Go
+* Last updated: 2024-04-18
+* Discussion at: 
+
+## Abstract
+
+This document proposes a design for a new load balancing policy called pid. The term pid stands for [Proportional–integral–derivative controller](https://en.wikipedia.org/wiki/Proportional%E2%80%93integral%E2%80%93derivative_controller). This policy builds upon the [A58: weighted_round_robin LB policy (WRR)][A58] and requires direct load reporting from backends to clients. Similar to wrr, it utilizes client-side weighted round robin load balancing. However, unlike wrr, it does not determine weights deterministically. Instead, it employs a feedback loop with the pid controller to adjust the weights in a manner that allows the load on all backends to converge to the same value. The policy supports either per-call or periodic out-of-band load reporting as per [gRFC A51][A51].
+
+## Background
+
+The `wrr` policy uses the following formula to calculate subchannel weights::
+
+$$weight = \dfrac{qps}{utilization + \dfrac{eps}{qps} * error\\_utilization\\_penalty}$$
```

Comment:
might be worth just linking directly to the `wrr` RFC which goes into the details here.

Reply:
added.